R/count_labels.R

```diff
@@ -133,8 +137,7 @@ countBar <- function(inputId, label, countmax, countnow = NULL, counttotal = cou
 }
 
 #' @rdname countBars
-updateCountBars <- function(session = getDefaultReactiveDomain(), inputId, choices, # nolint
-                            countsmax, countsnow = NULL) {
+updateCountBars <- function(session = getDefaultReactiveDomain(), inputId, choices, countsmax, countsnow = NULL) { # nolint
```

Comment:
for new nolints - would you mind specifying which lintr? (obviously not asking to fix all existing ones)
Here it's most likely `object_name_linter`

Reply:
hmm that's actually not a new `nolint` - sorry

Reply:
This one is for camelCase in variable name, which is in keeping with related `shiny` functions.